webpack.config.js

```diff
@@ -252,6 +252,16 @@ module.exports = (env, argv) => {
                         outputPath: '.',
                     },
                 },
+                {
+                    test: /encoderWorker\.min\.js$/,
+                    loader: "file-loader",
+                    type: "javascript/auto", // https://github.com/webpack/webpack/issues/6725
+                    options: {
+                        // We deliberately override the name so it makes sense in debugging
+                        name: 'opus-encoderWorker.min.[hash:7].[ext]',
```

Comment:
Huh? I don't think I am following... Why not just name it `opus-...` from the start...?

Reply:
Oh, I guess it's from an NPM package.

Reply:
have added docs anyhow for the next person who reads this